surrealdb/clients/http.py

```diff
@@ -110,43 +130,123 @@ async def _request(
         return response_obj
 
     async def execute(self, query: str) -> List[Dict[str, Any]]:
+        """Executes a query against the SurrealDB server.
+
+        Parameters
+        ----------
+        query: :class:`str`
+            The query to execute.
+
+        Returns
+        -------
+        :class:`List[Dict[str, Any]]`
+            The results of the query.
+        """
         response = await self._request(method="POST", uri="/sql", data=query)
         return response.result
 
     async def create_all(self, table: str, data: Any) -> List[Dict[str, Any]]:
+        """Creates multiple items in a table.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table to create the item in.
+        data: :class:`Any`
+            The data to insert into the table.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The items created
+        """
         response = await self._request(
             method="POST",
             uri=f"/key/{table}",
             data=jsonlib.dumps(data),
         )
-
         return response.result
 
     async def create_one(self, table: str, id: str, data: Any) -> Dict[str, Any]:
+        """Creates a single item in a table.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table to create the item in.
+        id: :class:`str`
+            The id of the item to select.
+        data: :class:`Any`
+            The data to insert into the table.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The item created.
+        """
         response = await self._request(
             method="POST",
             uri=f"/key/{table}/{id}",
             data=jsonlib.dumps(data),
         )
-
         return response.result[0]
 
     async def select_all(self, table: str) -> List[Dict[str, Any]]:
+        """Selects all items in a table.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table to select from.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            All items retrieved from the table.
+        """
         response = await self._request(method="GET", uri=f"/key/{table}")
         return response.result
 
     async def select_one(self, table: str, id: str) -> Dict[str, Any]:
+        """Selects a single item in a table.
+
+        Parameters
+        ----------
+        table: :class:`str`
+            The table to select from.
+        id: :class:`str`
+            The id of the item to select.
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The item selected.
+        """
         response = await self._request(method="GET", uri=f"/key/{table}/{id}")
         if not response.result:
             raise SurrealException(f"Key {id} not found in table {table}")
-
```

Comment:
why are we removing these whitespaces? it separates out the logic and removing them is harder to read